src/pages/blog/side-cta/_style.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components'
+import Shape from './images/shape.svg'
+import { Flex } from 'components/containers'
+import device from 'themes/device'
+import { LinkButton } from 'components/form'
+
+export const SideContainer = styled(Flex)`
+    padding: 20px;
+    justify-content:start;
+
+    @media ${device.tablet} {
+        justify-content:center;
+    }
+`
+export const CtaContainer = styled(Flex)`
+    width: 282px;
+    height: 420px;
+    border-radius: 8px;
+    padding-top: 25px;
+    padding-left: 16px;
+    background-color: var(--color-grey-41);
+    flex-flow:column;
+    justify-content:flex-start;
+    background-image: url(${Shape});
+    background-position: right bottom; 
+    background-repeat: no-repeat;
+
+    @media ${device.tablet} {
+        width: 328px;
```

Comment:
```suggestion
        width: 100%;
        max-width: 328px;
```

Reply:
You probably wanna do something like this instead to cater all the way down to 320px.